drivers/drivers.go

```diff
@@ -175,3 +175,10 @@ func GetSSHCommandFromDriver(d Driver, args ...string) (*exec.Cmd, error) {
 
 	return ssh.GetSSHCommand(host, port, user, keyPath, args...), nil
 }
+
+// SetVersioner is an interface to be implemented by specific Drivers that need access to the
+// current version of docker-machine. If a driver implements SetVersioner, it will be called with
+// the version on initialization.
+type SetVersioner interface {
+	SetVersion(string)
```

Comment:
This is a bit of an awkward way to get the `VERSION` constant into the driver. I can't reference it directly from the `main` package because it creates an import cycle. If there's a cleaner way to accomplish this that I'm not seeing, then I'm all ears :smile: